app/views/search/index.html.erb

```diff
@@ -0,0 +1,7 @@
+Station name: <%=  @station.name %><br>
+Address: <%=  @station.full_address %><br>
+Fuel type: <%=  @station.fuel_type %><br>
+Access times: <%= @station.access_time %>
+<br><br>
+Distance to final point: <%= @directions.distance %> mi<br>
+Travel time: <%= @directions.travel_time %> min
```

Comment:
Just to follow up on the comment I left on the Directions model -- Instead of calling the travel_time attribute here, call an instance method on the Directions model that will do that calculation for you.